server/helpers/utils/utils.go

```diff
@@ -262,3 +268,14 @@ func SliceContains(elements []string, name string) bool {
 	}
 	return false
 }
+
+func GetPlatform() (platform string) {
+	platform = "Unsupported Platform"
+	_, err := os.Stat("/var/run/docker.sock")
```

Comment:
This is very fragile. Some old kubernetes version have this mounted so this would not work. 
Plus it is not necessary that docker socket is at this path. In docker desktop it is, but in a lot of other solutions socket can be anywhere. We can't rely on this. 
First make the check for kubernetes before docker. And also stat for `/var/run/secrets/kubernetes.io/serviceaccount` in case of kubernetes.

Reply:
How about now?